src/client/pages/Checkout/CheckoutPayment/ContactInformation/ContactInformation.tsx

```diff
@@ -10,13 +10,14 @@ import { MEDIUM_SMALL_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 import { Divider } from '../../shared/Divider'
 
 export type ContactInfoData = {
-  firstName?: string
-  lastName?: string
-  ssn?: string
-  email?: string
+  firstName?: string | null | undefined
+  lastName?: string | null | undefined
+  ssn?: string | null | undefined
+  email?: string | null | undefined
 }
```

Comment:
Isn't this just exposing strange things from the API? I think we should avoid letting it through to the React components. Let's sort that out in the hook :)

Seems like in this component we only want defined strings. So just type it that way - we should not care about API stuff here.